### PR TITLE
Migrate to Node.js 16 LTS and update dependencies

### PR DESCRIPTION
**Pull request by iby**
Also use strict assert instead of deprecated legacy mode to avoid the following warning, also recommended in official
[documentation](https://nodejs.org/docs/v16.13.2/api/assert.html#legacy-assertion-mode):

```
TS1259: Module '"assert"' can only be default-imported using the 'esModuleInterop' flag
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,10 +23,10 @@
   ],
   "dependencies": {},
   "devDependencies": {
-    "@types/node": "^14",
-    "browserify": "^16.5.1",
-    "loltest": "github:lookback/loltest#1.1.1",
+    "@lookback/loltest": "^1",
+    "@types/node": "^16",
+    "browserify": "^17",
```
Comment (joakim-lookback) [inline package.json:28]: This was unimaginable just a few months ago. Great work getting us to this point @iby!

**Review by joakim-lookback — APPROVED** [review]
LGTM!

File: package.json (hunk 1 above)
Comment (iby) [reply package.json:28]: What? Why? Really? What am I missing?

**Comment by joakim-lookback** [comment]
@iby Where there any particular changes that made you nervous? The changes in dependencies seemed fine. I bet you tried building on browserify 17.

File: package.json (hunk 1 above)
Comment (joakim-lookback) [reply package.json:28]: Oh wow! Now I mixed things up. You upgraded gulp in the chrome ext, not browserify here 😅 Never mind me. Just take the credit god damn it!!

**Comment by iby** [comment]
@joakim-lookback Yep. It builds and tests are all passing. Looks all good from the code point of view. I never touched this repo and was wondering if we're okay with browserify 17 here – just being overly careful subconsciously.

**Comment by joakim-lookback** [comment]
@iby I haven't worked with this repo either, so if you want to be 100% sure you didn't screw something up then I think you need to wait for a review from @algesten.

**Pull request merged**